script/age_grouping.R

```diff
@@ -0,0 +1,83 @@
+## Age Grouping
+#Preparations
+library(tidyverse)
+
+clean_data <- read.csv(here::here("I:/pwind/Nextcloud/Datascience/OceanNow_data/clean_data.csv"))
```

Comment:
You can store the data in /data folder within the repo and then use a relative path. I've written the .gitignore file to ignore the /data folder, so the data won't be made public.

```suggestion
clean_data <- read.csv(here::here("./data/clean_data.csv"))
```